### PR TITLE
Alert & Button

### PR DESCRIPTION
**Pull request by manojncsoni007**
completed alert and button componen
Screenshot (168)
Screenshot (169)
t

### CODE REVIEW

File: Components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        background-color: var(--red);
+        color: var(--white);
+        border-radius: 50%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+    
+    .floating-btn:hover {
+        background-color: #df2727;
```
Comment (HarshithaKonduru214) [inline Components/button/button.css]: Use variables instead of hex codes.

File: Components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        flex-direction: row;
+        justify-content: flex-start;
+    }
+    
+    .icon-btn:hover {
+        transform: scale(1.2);
+    }
+    
+    .link-btn {
+        background-color: #df2727;
```
Comment (HarshithaKonduru214) [inline Components/button/button.css]: Use variables instead of hex codes.

**Review by HarshithaKonduru214 — APPROVED** [review]

**Comment by som749898** [comment]
looks good

**Review by anupkgurung — APPROVED** [review]
seems good.

**Pull request merged**